gluten-substrait/src/main/scala/org/apache/gluten/execution/GlutenWholeStageColumnarRDD.scala

```diff
@@ -36,7 +36,9 @@ case class GlutenPartition(
     index: Int,
     plan: Array[Byte],
     splitInfosByteArray: Array[Array[Byte]] = Array.empty[Array[Byte]],
-    locations: Array[String] = Array.empty[String]
+    locations: Array[String] = Array.empty[String],
+    files: Array[String] =
+      Array.empty[String] // touched files, for implementing UDF input_file_name
```

Comment:
Is this a relevant change? If not, better to include this change in another pr.

Reply:
Yes. It is used to implement function like `select input_file_name() from range();`